src/services/api/api-base.js

```diff
@@ -75,10 +76,32 @@ export function tryCastToApiError(error) {
   throw error
 }
 
-export function mapObjectId(id){
+export function toUrlParam(value) {  
+  if (value instanceof(Date))  return value.toISOString();
+  if (value instanceof(Object)) return JSON.stringify(value);  
+  return value; 
```

Comment:
I think this can be simplied to this:

```suggestion
  if(value!==undefined) value = JSON.stringify(value);
  return value; 
```

Reply:
if value is number or string ,  use JSON.stringify(value) will cause error.